### DISCUSSION TITLE
Github 介面上沒有 commit branch releases contributor 那一列快捷鍵，該如何調整出來

### DISCUSSION THREAD

**Discussion by chengcheng1231** [Lidemy/forum#55] [其他問題]
問題如標題，以下圖片顯示我的介面未有那一列快捷鍵，因此沒辦法在這個 branch 下看 commit 有幾版了
![image](https://user-images.githubusercontent.com/36998811/115143080-c87b2c80-a077-11eb-9345-023a48da5aa6.png)

但應該是要有的如老師的介面
![image](https://user-images.githubusercontent.com/36998811/115143116-fbbdbb80-a077-11eb-83a3-cd3a61efc6da.png)

**Comment by cwc329** [comment]
根據我的猜測，
應該是網頁改版過所以介面有換。
![截圖](https://i.imgur.com/SCiDlLv.png)
這是目前的頁面，左邊可以看到 'master' 這就是可以選擇 branch，
branch 選項下面可以看到 'github-classroom initial commit' 的訊息，
這行訊息最右邊就可以看到現在有幾個 commit 了。

**Comment by cwc329** [comment]
因為 huli 在錄這個影片的時候應該離現在有段時間，
網頁改版會介面也是很正常的事情。
